exasol_integration_test_docker_environment/lib/base/ssh_access.py

```diff
@@ -92,18 +94,17 @@ def generate(cls) -> 'SshKey':
         return SshKey(rsa_key)
 
     @classmethod
-    def from_folder(cls, folder: Optional[Path] = None) -> 'SshKey':
-        files = SshFiles(folder)
-        priv = files.private_key
-
+    def from_files(cls, ssh_files: Optional[SshFiles] = None) -> 'SshKey':
```

Comment:
maybe, we should rename this to new_from_files

Reply:
renamed to `from_cache()`. As it is a class method I would prefer to not add the prefix `new_`.